Samples~/ThirdPersonDemoURP/Demo/Scripts/EnemyController.cs

```diff
@@ -129,6 +131,7 @@ public Sprite GetIcon()
         /*
          * Gets the enemies current hp
          */
+        [RGState("currentHealth")]
```

Comment:
We're mixing typecase. These states begin with a lowercase letter, but `IsAttacking` starts with an uppercase. Let's make sure they're consistent

Reply:
Fixed